src/net/libwebsocketsIO.cpp

```diff
@@ -355,6 +361,11 @@ int LibwebsocketsClient::wsCallback(struct lws *wsi, enum lws_callback_reasons r
             {
                 X509_STORE_CTX_set_error((X509_STORE_CTX*)user, X509_V_OK);
             }
+            else
+            {
+                X509_STORE_CTX_set_error((X509_STORE_CTX*)user, X509_V_ERR_UNSPECIFIED);
```

Comment:
Maybe it's better to use a more specific error code here, since probably OpenSSL will use this `X509_V_ERR_UNSPECIFIED` for other reasons.
I think that `X509_V_ERR_APPLICATION_VERIFICATION` could be a good one.
> application verification failure 
>    an application specific error. This will never be returned unless explicitly set by an application.